system/lib/libc/musl/src/thread/pthread_cancel.c

```diff
@@ -66,6 +67,7 @@ static void cancel_handler(int sig, siginfo_t *si, void *ctx)
 
 	__syscall(SYS_tkill, self->tid, SIGCANCEL);
 }
+#endif
 
 void __testcancel()
```

Comment:
If we are including this function, then perhaps we should also be including `system/lib/libc/musl/src/thread/pthread_testcancel.c` rather than our custom `system/lib/pthread/pthread_testcancel.c`?

How about `ifdef`'ing this function out for now and the adding it in as followup PR which includes `system/lib/libc/musl/src/thread/pthread_testcancel.c`?  (i.e. lets keep this PR small and focused.. but we can remove these unused functions).

Reply:
Ah you're right, I forgot to adjust the `ifdef`'s when I split PR #15604 (which includes `pthread_testcancel` from musl rather than using a custom one). Fixed with commit f4058e2e8aa4fdda6aed317327d654b898371c2a.